packages/models/proto/v1/eservice/eservice.proto

```diff
@@ -11,6 +11,34 @@ message EServiceV1 {
   optional EServiceAttributesV1 attributes = 6;
   repeated EServiceDescriptorV1 descriptors = 7;
   optional int64 createdAt = 8;
+  repeated EServiceRiskAnalysisV1 riskAnalysis = 9;
```

Comment:
This is how it is now in the Scala codebase and already in production, we confirmed this in a Slack thread with @galales - it is not causing issues since protobuf handles repeated as optional and maps them directly to `[ ]`